src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
nitpick : might need more shadows or something in the horitonzal menus of the toolbar, it kinda melds with the bottom border 
![image](https://user-images.githubusercontent.com/31045223/221854414-60e3ecbc-66a5-485d-b386-b2afac6c4b02.png)